src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -880,8 +882,11 @@ int InputProcessor::getObjectItemNum(std::string const &ObjType, // Object Type
     }
 
     if (!found) {
-        // ShowWarningError("Didn't find name, need to probably use search key");
-        return -1;
+        if (objectTypeFound) {
+            return 0;
+        } else {
+            return -1;
+        }
```

Comment:
Looking at this again I don't think any of this is needed. At line 865 a -1 is returned if the object type is not found. So if it makes it this far, it must be the name that is not found so at line 884  return 0 instead of -1.